ts/Extensions/Annotations/Types/Measure.ts

```diff
@@ -498,22 +503,20 @@ class Measure extends Annotation {
                 dashStyle: 'Dash',
                 overflow: 'allow',
                 align: 'left',
-                vertical: 'top',
+                y: 0,
+                verticalAlign: 'top',
                 crop: true,
-                point: function (target: any): PositionObject {
-                    const annotation: Measure = target.annotation,
-                        chart = annotation.chart,
-                        inverted = chart.inverted,
-                        xAxis = chart.xAxis[typeOptions.xAxis],
-                        yAxis = chart.yAxis[typeOptions.yAxis],
-                        top = chart.plotTop,
-                        left = chart.plotLeft;
+                xAxis: 0,
+                yAxis: 0,
+                point: function (target: any): ExtendedPositionObject {
+                    const annotation: Measure = target.annotation;
+                    const options = target.options;
```

Comment:
Would be nice to stick with our pattern of declaring multiple variables with only one `const` key word:
```js
const annotation: Measure = target.annotation,
    options = target.options;
```